<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Dome</class>
 <widget class="QDialog" name="X2Dome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>712</width>
    <height>548</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>712</width>
    <height>548</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RTI-Zone Dome controller settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>1024</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="SerialPortName" stdset="0">
      <string/>
     </property>
     <widget class="QGroupBox" name="MaxDomeIIParams">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>96</y>
        <width>296</width>
        <height>224</height>
       </rect>
      </property>
      <property name="title">
       <string>Dome params</string>
      </property>
      <widget class="QCheckBox" name="homeOnUnpark">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>193</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Home after unpark</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>29</y>
         <width>281</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hasShutterCtrl">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>151</y>
         <width>183</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter controller present</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="needReverse">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>130</y>
         <width>132</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reverse direction</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="homeOnPark">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>172</y>
         <width>136</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Home before park</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>64</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Position (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="homePosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>64</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>96</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Park Postition (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="parkPosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>96</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>336</x>
        <y>16</y>
        <width>336</width>
        <height>440</height>
       </rect>
      </property>
      <property name="title">
       <string>Controller settings</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>336</y>
         <width>152</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Dome battery cutoff :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lowRotBatCutOff">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>336</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <double>11.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>368</y>
         <width>152</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter battery cutoff :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lowShutBatCutOff">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>368</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <double>11.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>64</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation speed (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotationSpeed">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>64</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotationAcceletation">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>96</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>96</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation acceleration (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>34</y>
         <width>208</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of steps per revolution :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="ticksPerRev">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>34</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterAcceleration">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>200</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>200</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter acceleration (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>168</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter speed (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterSpeed">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>168</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>272</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter watchdog timer (s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterWatchdog">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>272</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>60</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>304</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rain sensor check interval (s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="rainCheckInterval">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>304</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>128</y>
         <width>264</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore dome motor default settings</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>232</y>
         <width>264</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore shutter motor default settings</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>8</y>
        <width>298</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>RTI-Zone.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>RTI-Zone.png</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>472</x>
        <y>488</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>576</x>
        <y>488</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>328</y>
        <width>296</width>
        <height>104</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="rainStatus">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>72</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>24</y>
         <width>136</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Dome battery level :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="shutterBatteryLevel">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>48</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>12.25 V</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>72</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rain sensor :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="domeBatteryLevel">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>24</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>12.25 V</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>48</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter battery level :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>440</y>
        <width>296</width>
        <height>72</height>
       </rect>
      </property>
      <property name="title">
       <string>Wireless link to shutter</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>32</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>PAN ID (hex) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="panID">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>32</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="prefix">
        <string>0x</string>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>16962</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>32</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
